management/sql/client.go

```diff
@@ -141,6 +169,28 @@ func (c SqlDatabaseClient) ListDatabases(server string) (ListDatabasesResponse,
 	return databases, err
 }
 
+// Updates details of the given SQL Database off the given server.
+//
+// https://msdn.microsoft.com/en-us/library/azure/dn505718.aspx
+func (c SqlDatabaseClient) UpdateDatabase(
+	server, database string,
+	params ServiceResourceUpdateParams) (management.OperationID, error) {
+	if database == "" {
+		return "", fmt.Errorf(errParamNotSpecified, "database name")
+	}
+	if server == "" {
+		return "", fmt.Errorf(errParamNotSpecified, "server name")
+	}
+
+	url := fmt.Sprintf(azureUpdateDatabaseURL, server, database)
+	req, err := xml.Marshal(params)
+	if err != nil {
+		return "", err
+	}
+
+	return c.mgmtClient.SendAzurePutRequest(url, "", req)
```

Comment:
I can't see it in the docs, but I think the Content-Type probably should be text/xml.